src/components/attendee/VideoComms/types.ts

```diff
@@ -0,0 +1,97 @@
+import Twilio from "twilio-video";
+
+import { Branded } from "utils/types";
+
+export type TrackId = Branded<string, "TrackId">;
+export type ChannelId = Branded<string, "ChannelId">;
+
+export enum VideoCommsStatus {
+  Disconnected = "DISCONNECTED",
+  Connecting = "CONNECTING",
+  Connected = "CONNECTED",
+  Errored = "ERRORED",
+}
+
+export enum VideoSource {
+  Webcam = "WEBCAM",
+  Screenshare = "SCREENSHARE",
+}
+
+export type VideoTrack = {
+  id: TrackId;
+  attach: (el: HTMLVideoElement) => void;
+  detach: () => void;
+  kind: "video";
+  twilioTrack: Twilio.VideoTrack;
+  sourceType: VideoSource;
+  enabled: boolean;
+};
+
+export type AudioTrack = {
+  id: TrackId;
+  attach: (el: HTMLAudioElement) => void;
+  detach: () => void;
+  kind: "audio";
+  twilioTrack: Twilio.AudioTrack;
+  enabled: boolean;
+};
+
+export type Track = VideoTrack | AudioTrack;
+
+export interface Participant {
+  /**
+   * ID of the participant - used for communication with the video platform
+   */
+  id: string;
```

Comment:
Should this be `id`? if so, comment is a bit useless. Might reword like:
` id used for communication with twillio, @see` followed by documentation URL